### PR TITLE
Update nodejs branch

### PR DESCRIPTION
**Pull request by MyDapitt**
### All Submissions:

* [x] Have you ensured there aren't other open [Pull Requests](../pulls) for the same update or change?
* [ ] Have you created a new branch for your changes and PR from that branch and not from your master branch?

### New Image Submissions:

1. [x] Have you added your image to the [Github workflows](https://github.com/parkervcp/yolks/tree/master/.github/workflows)?
2. [x] Have you updated the README list to contain your new image?

### CODE REVIEW

File: nodejs/18/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN         apt update \
             && apt -y install ffmpeg iproute2 git sqlite3 libsqlite3-dev python3 python3-dev ca-certificates dnsutils tzdata zip tar curl build-essential libtool iputils-ping libnss3 tini \
             && useradd -m -d /home/container container
 
-RUN         npm install npm@9.8.1 typescript ts-node @types/node --location=global
+RUN         npm install npm@10.x.x typescript ts-node @types/node --location=global
```
Comment (QuintenQVD0) [inline nodejs/18/Dockerfile:9]: 9.8.1 is the latest supported for nodejs 18?

File: nodejs/20/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM        --platform=$TARGETOS/$TARGETARCH node:20-bookworm-slim
+FROM        --platform=$TARGETOS/$TARGETARCH node:20-slim
```
Comment (QuintenQVD0) [inline nodejs/20/Dockerfile:1]: keep this to bookworm, unless you have a real valid reson to not.

We lock it for a reson as we had it brake in the past with os updates (11->12)

File: nodejs/22/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+FROM        --platform=$TARGETOS/$TARGETARCH node:22-slim
```
Comment (QuintenQVD0) [inline nodejs/22/Dockerfile:1]: no need for 22 as it will be EOl in 5 months

File: nodejs/23/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,25 @@
+FROM        --platform=$TARGETOS/$TARGETARCH node:23-slim
```
Comment (QuintenQVD0) [inline nodejs/23/Dockerfile:1]: bookwurm please

File: nodejs/18/Dockerfile (hunk 1 above)
Comment (MyDapitt) [reply nodejs/18/Dockerfile:9]: No, the new version of Nodejs 18 is 18.20.4. It can install npm v10.x.x

File: nodejs/23/Dockerfile (hunk 4 above)
Comment (MyDapitt) [reply nodejs/23/Dockerfile:1]: Wait a minutes..

Comment (MyDapitt) [reply nodejs/23/Dockerfile]: Ok, i was updating the images base

**Comment by MyDapitt** [comment]
How, is my request approved?

**Comment by QuintenQVD0** [comment]
> How, is my request approved?

I will look later on this day. It is 7:30 AM for me.

**Comment by MyDapitt** [comment]
> > How, is my request approved?
> 
> I will look later on this day. It is 7:30 AM for me.

Ok

**Review by QuintenQVD0 — APPROVED** [review]
Seems Fine to me now

**Comment by MyDapitt** [comment]
Are the changes I made propagated to the main repository?

**Comment by QuintenQVD0** [comment]
> Are the changes I made propagated to the main repository?

Not yet. There aproved. Not yet merged.

**Pull request merged**